browser/main/lib/dataApi/attachmentManagement.js

```diff
@@ -253,23 +261,57 @@ function generateAttachmentMarkdown (fileName, path, showPreview) {
  * @param {Event} dropEvent DropEvent
  */
 function handleAttachmentDrop (codeEditor, storageKey, noteKey, dropEvent) {
-  const file = dropEvent.dataTransfer.files[0]
-  const filePath = file.path
-  const originalFileName = path.basename(filePath)
-  const fileType = file['type']
-  const isImage = fileType.startsWith('image')
-  let promise
-  if (isImage) {
-    promise = fixRotate(file).then(base64data => {
-      return copyAttachment({type: 'base64', data: base64data, sourceFilePath: filePath}, storageKey, noteKey)
+  if (dropEvent.dataTransfer.files.length > 0) {
```

Comment:
Intuitively i would say this if-condition is exactly the wrong way around.. If there is **more then 0 Attachments**, only take the first one, otherwise iterate over all (but otherwise means that there are zero attachments, wouldn't it?!)

Reply:
I have the PR #2473 which add the loop. I could add it here.